src/main/java/mate/academy/service/DriverService.java

```diff
@@ -13,4 +14,6 @@
     Driver update(Driver driver);
 
     boolean delete(Long id);
+
+    Optional<Driver> findByLogin(String login);
```

Comment:
If I'm right and we should inject the dao to AuthenticationService then service should return Driver and not Optional of Driver